### ISSUE TITLE
3d printer

### ISSUE THREAD

**Issue by mitar** [cloyne/hackerspace#37]
Let's buy a 3D printer. I would suggest that for the first 3D printer we buy this one: http://robo3d.com/

It is cheap, it is reportedly easy to use (@SloMusti says so), and well supported.

We can complicate later on. Let's buy this as soon as possible so that people can start learning how to use it. We do not have even to wait for the makerspace computer.

**Comment by sunnysideofthescreen** [comment]
@emilynathan is this included in the current makerspace budget? if not, we would have to pass it through next council separately

**Comment by mitar** [comment]
Yes, it was included already. So let's do it now, because otherwise we will have to pass it again next semester!

**Comment by jmrtns** [comment]
Alright, I'll buy it off of Amazon?

**Comment by jmrtns** [comment]
I'll buy a roll of filament too. What color?

**Comment by rmilman** [comment]
Please hold off for a bit!

**Comment by sunnysideofthescreen** [comment]
Emily is reworking their discretionary budget and plans to purchase
everything this week (fingers crossed)

**Comment by jmrtns** [comment]
Cool, let me know if you need me to buy anything, I can easily front a few thousand.

**Comment by sunnysideofthescreen** [comment]
good to know!

**Comment by ahdinosaur** [comment]
heya folx. :cat2: what about using this as an opportunity to support our local 3d printer businesses (such as [Type A Machines](http://www.typeamachines.com/)) and / or open source 3d printers (such as [the Rep Rap family](http://www.reprap.org/))?

**Comment by sunnysideofthescreen** [comment]
hiya there :) i agree this is a sweet opportunity to use our 3d printer
money wisely. thanks for the recommendations :) Those Type As are beautiful
but I think are out of our budget (~1300?). Reprap is cool looking too. i
think the person with the $$ is looking at the printrbot,
http://printrbot.com/. Any comments on this machine?
.

**Comment by ahdinosaur** [comment]
awesome. :) recommend buying something built with quality (and preferably also hackable ;), i find local and open source printers are usually good at both if you pay someone to build it for you.

the latest printrbots seem awesome, it seems just yesterday they were funded on Kickstarter, if i was buying a printer for the house i'd do it.

many more options [here](http://www.reprap.org/wiki/RepRap_Buyers%27_Guide#RepRap_shops) and [here](https://www.reddit.com/r/3Dprinting/wiki/printerchart), but :shipit:.

**Comment by sunnysideofthescreen** [comment]
so many to choose from :P pretty exciting.

feel free to come by and play when we finally get it :) if you have much
expertise with the printer, it would be great to have ya teach some house
folks how to use/play-with it!

-kel

**Comment by SloMusti** [comment]
I strongly advise if you do not have people at hand who are willing to spend time and effort taking care of the 3d printer and teaching everyone else to get one that is plug&play. Open source printers generally fail from this perspective and require some fiddling with.

**Comment by emilynathan** [comment]
Hiya! We decided on the [standard printrbot] (http://printrbot.com/shop/assembled-simple-metal-with-heated-bed/?attribute_pa_asm-simple-mtl-w-heated-bed=assembled-simple-with-heated-bed-black) with heated bed. We had some raving reviews from 3DMC, Cal's 3D printing club. They called it dependable, durable, and easy to start with, which I think will be perfect for the coop! It also uses [free software] (https://ultimaker.com/en/products/cura-software/list) that's compatible with MAC OS, Linux, and Windows!

They also sell filament which runs at market price (20-30 dollars per kg for PLA or ABS) in 0.5 kg spools. We figured that we might as well buy a few spools of PLA to get ourselves started (the printer is compatible with both PLA and ABS). We were thinking of getting one spool of the [black] (http://printrbot.com/shop/ultra-black-pla/?attribute_pa_filament=half-kg), [white] (http://printrbot.com/shop/snow-white-pla/?attribute_pa_filament=half-kg), and [GLOW IN DA DARK~] (http://printrbot.com/shop/ghost-green-pla/?attribute_pa_filament=half-kg). 

This will bring our total purchase to almost exactly $850, which is significantly under budget!

**Comment by emilynathan** [comment]
We initially wanted to purchase a lulzbot, but heard that it had a tendency to breakdown. We were also looking into Makerbots, but they tended to run a bit expensive. We looked into the Mendelmaker family as well because we were considering building it ourselves, but eventually decided that getting a buy n go might be better, as this purchase has been heavily delayed (for good reason though!).

Thank you all for your input!

**Comment by sunnysideofthescreen** [comment]
yeeeeewwwwwww!

**Comment by emilynathan** [comment]
Ye~~~ We can also ask 3DMC to come in and help us if we have troubleshooting issues, and they may even want to give a workshop in the future!

**Comment by ahdinosaur** [comment]
(ﾉ´ヮ´)ﾉ*:･ﾟ✧

**Comment by jmrtns** [comment]
The printer and all the filament should arrive on Tuesday.

**Comment by sunnysideofthescreen** [comment]
Yewww. Winter break mischief ensues.

On Dec 13, 2015, at 12:16 PM, jmrtns <notifications@github.com> wrote:

The printer and all the filament should arrive on Tuesday.

On Wed, Dec 9, 2015 at 10:56 PM, Mikey <notifications@github.com> wrote:

> (ﾉ´ヮ´)ﾉ*:･ﾟ✧
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloyne/hackerspace/issues/37#issuecomment-163518428>.
>

—
Reply to this email directly or view it on GitHub
<https://github.com/cloyne/hackerspace/issues/37#issuecomment-164294559>.

**Comment by mitar** [comment]
Don't burn the house while I am gone. ;-)

**Comment by ahdinosaur** [comment]
obligatory Cloyne quote:

!["Some say Cloyne will end in fire. Others, also in fire."](https://cloud.githubusercontent.com/assets/719605/11773042/37f6bea0-a289-11e5-8079-9379c64d62ba.jpg)

(hope y'all still have this gem of a mural :wink:)

**Comment by mitar** [comment]
@ahdinosaur: You do not even know ... we just recently have not one, but two fires.

**Comment by mitar** [comment]
> (hope y'all still have this gem of a mural :wink:)

Of course. :-)

BTW, do you have an archive of photos of all old murals somewhere? We wanted to do make a site with history of all murals for some time now.

**Comment by sunnysideofthescreen** [comment]
:)




​

**Comment by ahdinosaur** [comment]
@mitar hehe. i pulled this from [savecloyne.com](http://savecloyne.com/multimedia.html), there's a few videos and images of murals there from while we were being purged.

**Comment by mitar** [comment]
Aha. I thought that you made an archive? Didn't you? I heard that at that time. That old Clones made digital photos of all murals.

**Comment by ahdinosaur** [comment]
@mitar [:gift:](https://ocf.berkeley.edu/~mwilliams/misc/cloyne-house-spring-2014.zip).

**Comment by mitar** [comment]
:-)

**Comment by mitar** [comment]
I know that one should not criticize the gift, but those photos are soooo smaaal!

(BTW, others, this is how you hijack the issue for something completely unrelated and is frowned upon in open source.)

**Comment by ahdinosaur** [comment]
@mitar oops, [:fish:](http://static.dinosaur.is/cloyne-house-spring-2014.zip).

**Comment by emilynathan** [comment]
@mitar I thought you were criticizing the little gift emoji at first lul

**Comment by mitar** [comment]
Thanks! I will add them to our file server!

**Comment by sunnysideofthescreen** [comment]
3D printer is in, right @jmrtns ? no dents this time?

**Comment by jmrtns** [comment]
Gah fuck this shit. The control board is bad. I'm going to stop getting it
from amazon. I'll return this one and buy one from the official store.
Hopefully that one won't be broken...
